bloom-housing/bloom
# Remove unused phone number and email fields from HouseholdMembers

# Pull Request Template

## Issue Overview

This PR fixes #2130


## Description

This is a mergeable version of #3306 (I somehow messed up commit state there... git is hard, esp with forks)

## How Can This Be Tested/Reviewed?

To test this I:

* Ran the migration to make sure there were no errors

* reseeded the DB to make sure that the changes I made to the seed script didn't throw any exceptions (and also made sure that the migration applied cleanly)

* tested making and submitting an application and did some extra CRUD operations around the household members step to make sure that I didn't break anything there.

* clicked around in http://localhost:3100/docs/ to check api doc changes and make sure that I didn't break anything

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
